python/amici/sbml_import.py

```diff
@@ -452,8 +454,15 @@ def _gather_base_locals(self):
             if r.isSetId():
                 self.local_symbols[r.getId()] = _get_identifier_symbol(r)
 
+        # oo is sympy infinity
+        self.local_symbols['INF'] = sp.sympify('oo')
+        self.local_symbols['NaN'] = sp.nan
+
+        # rem function
+        self.local_symbols['rem'] = sp.Mod
```

Comment:
Should an exception be raised if these symbols exist in the model? e.g.:
```python
reserved = {
    'INF': sp.oo,
    'NaN': sp.nan,
    'rem': sp.Mod,
}

if reserved.keys() & list_of_sbml_ids:
    raise ...
# or if it would have already been added to local_symbols, can do
if reserved.keys() & local_symbols.keys():
    raise ...

local_symbols.update(reserved)
```

Reply:
Good call, unfortunately SBML seems to indeed allow such silliness ...